github/messages.go

```diff
@@ -163,6 +165,11 @@ func WebHookType(r *http.Request) string {
 	return r.Header.Get(eventTypeHeader)
 }
 
+// DeliveryID returns the unique delivery ID of webhook request r.
```

Comment:
Perhaps it's a good idea to link to the relevant GitHub API docs here?

```
// DeliveryID returns the unique delivery ID of webhook request r.
//
// GitHub API docs: https://developer.github.com/v3/repos/hooks/#webhook-headers
```

If you want to, you can add the same to docs of `WebHookType`.